dashboard/README.md

```diff
@@ -1,7 +1,23 @@
 # Dashboard!!
 
 - Home page
+- Artists page
 - Locations page
-- Subscribe to emails
+- Subscriptions page
+- Tags page
```

Comment:
I'm assuming a section addressing each page will be part of the finished `README.md`?

Reply:
Yes there will be.